cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -78,15 +81,17 @@ type indexArgs struct {
 func (o *indexArgs) BuildOptions() *build.Options {
 	rawConfig := map[string]string{}
 	if o.IndexOptions.RepoID > 0 {
-		// NOTE(keegan): 2020-08-13 This is currently not read anywhere. We are
-		// setting it so in a few releases all indexes should have it set.
 		rawConfig["repoid"] = strconv.Itoa(int(o.IndexOptions.RepoID))
 	}
 
 	if o.Priority != 0 {
 		rawConfig["priority"] = strconv.FormatFloat(o.Priority, 'g', -1, 64)
 	}
 
+	if o.Public {
+		rawConfig["public"] = "1"
+	}
+
```

Comment:
should I exclude this? No one reads this and it could become outdated, it seems safer to rely on the public.txt file? (On that note, priority above is also never read).